hepcrawl/utils.py

```diff
@@ -69,6 +71,14 @@ def collapse_initials(name):
     return name
 
 
+def split_fullname(author):
+    """Split an author name to surname and given names from the form 'foo, bar'."""
+    fullname = re.sub(r',', '', author).split()
```

Comment:
I would suggest splitting on the comma, if given, and you will have the correct split. If comma is not given, you can indeed then just take the first.